WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyRingItemViewModel.cs

```diff
@@ -27,6 +27,21 @@ public PrivacyRingItemViewModel(PrivacyRingViewModel parent, SmartCoin coin, dou
 		AmountText = $"{Coin.Amount.ToFormattedString()} BTC";
 		Unconfirmed = !coin.Confirmed;
 		Confirmations = coin.GetConfirmations();
+
+		Reference =
+			this switch
+			{
+				{ IsPrivate: true } => "Private",
+				{ IsSemiPrivate: true } => "Semi private",
+				{ IsNonPrivate: true } => "Non private",
+				_ => ""
+			};
+
+		Reference += " coin";
```

Comment:
IMO it should be coins here too.

```suggestion
		Reference += " coins";
```

Reply:
I think the current one is correct. If the segment meant a bunch of coins it would be ok to say `coins` but as a segment mean a coin here, I think `coin` is correct.

Reply:
@yahiheb I'll keep the original wording, due to the fact that Ring might show individual coins as segments, OR it might show entire pockets as segments, when the number of UTXOs exceed a certain threshold (currently 100)